File: 源码/EnjoyFix/app/src/main/java/com/enjoy/enjoyfix/MyApplication.java
package com.enjoy.enjoyfix;

import android.app.Application;
import android.content.Context;
import android.support.v7.app.AppCompatActivity;

import com.enjoy.patch.EnjoyFix;

import java.io.File;

/**
 * @author Lance
 * @date 2019-09-03
 */
public class MyApplication extends Application {

    @Override
    protected void attachBaseContext(Context base) {
        super.attachBaseContext(base);
        EnjoyFix.installPatch(this, new File("/sdcard/patch.jar"));// 加载


        ClassLoader classLoader = MainActivity.class.getClassLoader();
        ClassLoader classLoader1 = AppCompatActivity.class.getClassLoader();
        ClassLoader classLoader2 = Application.class.getClassLoader();

        ClassLoader classLoader3 = getClassLoader();
    }

}

File: android/MultiLanguages/app/src/main/java/com/joyy/multilanguages/MyApplication.kt
package com.joyy.multilanguages

import android.app.Application
import android.content.Context
import com.joyy.languages.MultiLanguages

/**
 * Time:2021/9/16 15:14
 * Author: flannery
 * Description:
 */
class MyApplication : Application() {

    override fun attachBaseContext(base: Context) {
        super.attachBaseContext(MultiLanguages.attachContext(base))
    }

    override fun onCreate() {
        super.onCreate()
        // 多语言注册回调信息等
        MultiLanguages.inits(this)
    }

}